var/run/namelist.input

```diff
@@ -0,0 +1,182 @@
+ &wrfvar1
+ PRINT_DETAIL_BE=false,
+ PRINT_DETAIL_OBS=false,
+ WRITE_GTS_OMB_OMA=true,
+ /
```

Comment:
The namelist.input file should be removed. I will add a readme file for chem_da in another PR.